python/ray/tune/utils/util.py

```diff
@@ -749,6 +749,36 @@ def validate_warmstart(parameter_names: List[str],
                 " do not match.")
 
 
+def get_current_node_resource_key() -> str:
+    """Get the Ray resource key for current node.
+    It can be used for actor placement.
+
+    If using Ray Client, this will return the resource key for the node that
+    is running the client server.
+
+    Returns:
+        (str) A string of the format node:<CURRENT-NODE-IP-ADDRESS>
+    """
+    current_node_id = ray.get_runtime_context().node_id.hex()
+    for node in ray.nodes():
+        if node["NodeID"] == current_node_id:
+            # Found the node.
+            for key in node["Resources"].keys():
+                if key.startswith("node:"):
+                    return key
+    else:
+        raise ValueError("Cannot found the node dictionary for current node.")
+
+
+def force_on_current_node(task_or_actor):
```

Comment:
can you add Args and Return docstring for this, since it shows up in the user-facing examples?

Reply:
https://ray--16034.org.readthedocs.build/en/16034/tune/api_docs/trainable.html#utilities Can you also make sure it shows up here?

Reply:
Good point, updated.